gst/tensor_decoder/tensordec.c

```diff
@@ -424,7 +426,6 @@ gst_tensordec_set_property (GObject * object, guint prop_id,
         silent_debug ("tensor_decoder plugin mode (%s)\n", temp_string);
         for (i = 0; i < TensorDecMaxOpNum; i++)
           retval &= _tensordec_process_plugin_options (self, i);
```

Comment:
IMO, retval is unnecessary (instead, add code to print log if _tensordec_process_plugin_options() returns false.

Reply:
I agree. The new commit is updated.